fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/ConcreteAvailability.java

```diff
@@ -51,6 +51,16 @@ public ConcreteAvailability(
     }
 
     @Override
+    public SimplifiedIntervalList getAvailableIntervals() {
+        return null;
+    }
+
+    @Override
+    public Set<TableName> getDataSourceNames(PhysicalDataSourceConstraint constraint) {
+        return dataSourceNames;
```

Comment:
I don't think this override is needed, since it looks like it does what the default would be.